components/RegistrationForm/__tests__/RegistrationForm.test.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { fireEvent, render, waitFor } from '@testing-library/react';
-import { networkErrorMessages, validationErrorMessages } from 'common/constants/messages';
+import { networkErrorMessages } from 'common/constants/messages';
 import createSnapshotTest from 'test-utils/createSnapshotTest';
 import mockUser from 'test-utils/mockGenerators/mockUser';
-import mockPassword from 'test-utils/mockGenerators/mockPassword';
+/* import mockPassword from 'test-utils/mockGenerators/mockPassword'; */
```

Comment:
commented out?

Reply:
Was making sure I didn't need it while re-writing `mockPassword()`. Will remove it now.